src/commands/synthetics/api.ts

```diff
@@ -38,6 +39,10 @@ export class EndpointError extends Error {
 }
 
 export const formatBackendErrors = (requestError: AxiosError<BackendError>) => {
+  if (!requestError.config) {
+    return `config unavailable\n${requestError.message}`
+  }
```

Comment:
Why this change? And why only for the syntehtics API client?

Reply:
It was to make TypeScript happy, as the `config` can now be `undefined`. But I agree, this adds a new possible error, so let's use `requestError.config?.baseURL` instead.

And only on Synthetics because it's the only place where we read properties from `AxiosError.config`

Reply:
Addressed in a1fb70da5b09aa69d7ac6b435a5421223cb338f7